### PR TITLE
zh-CN: update glossary/crawler

### PR DESCRIPTION
**Pull request by PassionPenguin**
#### Modification

- improvements on words arragement
- linted word `Web`

#### Deletion

- link removal

### CODE REVIEW

File: files/zh-cn/glossary/crawler/index.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+title: Web 爬虫
 slug: Glossary/Crawler
+l10n:
+  sourceCommit: ada5fa5ef15eadd44b549ecf906423b4a2092f34
 ---
 
 {{GlossarySidebar}}
 
-网络爬虫是一种能有条理并且系统化地浏览{{glossary("World Wide Web","Web")}}，以便收集网页数据的程序（通常也被称作“机器人”）。搜索引擎（例如谷歌，必应）利用爬虫来创建索引是网络爬虫最典型的使用案例。
+Web 爬虫是一种能有条理并且系统化地浏览 {{glossary("World Wide Web","Web")}} 以收集网页数据的程序（通常也被称作“机器人”）。Web 爬虫最典型的使用案例是搜索引擎（例如谷歌，必应）使用爬虫来创建索引。
```
Comment (JasonLamv-t) [inline files/zh-cn/glossary/crawler/index.md:10]:
```suggestion
**Web 爬虫**是一种能有条理并且系统化地浏览 {{glossary("World Wide Web","Web")}} 以收集网页数据的程序（通常也被称作“机器人”）。Web 爬虫最典型的使用案例是搜索引擎（例如谷歌，必应）使用爬虫来创建索引。
```

File: files/zh-cn/glossary/crawler/index.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 
 ## 参见
 
 - 维基百科上的 [Web 爬虫](https://zh.wikipedia.org/wiki/網路爬蟲)
-- [MDN Web 文档术语表](/zh-CN/docs/Glossary)
-
-  - {{Glossary("Search engine")}}
+- {{Glossary("Search engine")}}
```
Comment (JasonLamv-t) [inline files/zh-cn/glossary/crawler/index.md:15]:
```suggestion
- {{Glossary("Search engine")}}
- 维基百科上的 [Web 爬虫](https://zh.wikipedia.org/wiki/網路爬蟲)
```

**Review by JasonLamv-t — CHANGES_REQUESTED** [review]

File: files/zh-cn/glossary/crawler/index.md [hunk 3]
```diff
@@ -1,15 +1,15 @@
 ---
-title: Crawler
+title: Web 爬虫
 slug: Glossary/Crawler
+l10n:
+  sourceCommit: oops, thx:)
```
Comment (yin1999) [inline files/zh-cn/glossary/crawler/index.md:5]: What's this?

**Review by JasonLamv-t — APPROVED** [review]
LGTM. Thx! @PassionPenguin

**Pull request merged**